### PR TITLE
GH-57: add python array

### PR DESCRIPTION
**Pull request by quang-ph**

### CODE REVIEW

File: concepts/python/array.md [hunk 1]
```diff
@@ -1,55 +1,74 @@
...
+
+| Operation       | Creation | Insertion        | Accessing | Update | Deletion         | Searching |
+|-----------------|----------|------------------|-----------|--------|------------------|-----------|
+| Time Complexity | $O(1)$   | $O(1)$ or $O(n)$ | $O(1)$    | $O(n)$ | $O(1)$ or $O(n)$ | $O(n)$    |
+
+## Array implementation in Python
+
+```python
+# Creation of Array ($O(1)$)
+import array as arr
```
Comment (rain1024) [inline concepts/python/array.md:19]: What are different between `array` and `list`?

Can you simply use `list` in this senario?

Comment (rain1024) [reply concepts/python/array.md:19]: Interesting. https://www.upgrad.com/blog/python-array-vs-list-differences-use-cases/

File: concepts/python/array.md [hunk 2]
```diff
@@ -1,55 +1,74 @@
...
-|-----------------|---------------------|
-| Operation       | $O(log(n))$         |
-
 ## 🔗 Further Reading
 
-* yyyy, [Awesome Reference Link](Awesome Reference Link), awesome.reference
-* yyyy, [Awesome Reference Link](Awesome Reference Link), awesome.reference
-* yyyy, ▶️ [Awesome Reference Link](Awesome Reference Link), awesome.reference
+* [Python Arrays](https://www.geeksforgeeks.org/python-arrays/), geeksforgeeks.org
+* ▶️ [CS50 2020 - Lecture 2 - Arrays](https://www.youtube.com/watch?v=tI_tIZFyKBw&t=3009s&ab_channel=CS50), CS50
```
Comment (rain1024) [inline concepts/python/array.md:74]: Worthy mention:
https://docs.python.org/3.8/library/array.html
https://www.upgrad.com/blog/python-array-vs-list-differences-use-cases/

Add two items and your style check error would be fixed.

**Review by rain1024 — CHANGES_REQUESTED** [review]
Please fix the style-check error

File: concepts/python/array.md (hunk 1 above)
Comment (quang-ph) [reply concepts/python/array.md:19]: I find some information in this article that is not really correct. "This means that storing such details in an immutable data structure such as an array makes much more sense" -> I think an array is a mutable data structure cause we can update its values after initialization

File: concepts/python/array.md (hunk 2 above)
Comment (quang-ph) [reply concepts/python/array.md:76]: Thank you for your recommendation. I added the contents.

**Review by rain1024 — APPROVED** [review]

**Pull request merged**